.gitignore

```diff
@@ -0,0 +1,6 @@
+build/
+coverage/
+node_modules/
+cache/
+artifacts
+src/typess/
```

Comment:
nit: `src/types`

Also should note, you should add leading forward slashes (ex: `build/` => `/build/`) if you only want to match folders in the root directory. This `gitignore` config will match any directory / file names that are `build` / `coverage` / `node_modules` / etc. Probably not an issue though.